runtime/gc_base/ContinuationObjectList.cpp

```diff
@@ -45,6 +45,38 @@ MM_ContinuationObjectList::MM_ContinuationObjectList()
 	_typeId = __FUNCTION__;
 }
 
+MM_ContinuationObjectList *
+MM_ContinuationObjectList::newInstance(MM_EnvironmentBase *env, uintptr_t listCount)
```

Comment:
Think it would be even more cleaner if the API is called more specifically newArrayInstances (or newInstanceArray..), and also rename the parameter something like arrayElements. Even put a comment in declaration that this API tries to mimic operator new[].

I want to avoid a potential misinterpretation of this API instantiating just one object and listCount being just an ordinary parameter to its constructor (that affects the size/shape of that one instance).

It would be nice if C++ would have allowed to return type MM_ContinuationObjectList[], but it did not (although probably for some valid reason).